internal/supervisor/import.go

```diff
@@ -88,16 +81,52 @@ func (s *ImportSupervisor) supervise() error {
 			continue
 		}
 
-		// if the State is stable and the Installation is slightly old, we
-		// can safely assume that an import happened and was completed
-		if installation.State == "stable" {
-			i.CompleteAt = time.Now().Unix() / 1000
+		if startedImportIsComplete(installation, i) {
+			i.CompleteAt = model.Timestamp()
 			err := s.store.UpdateImport(i)
 			if err != nil {
-				s.logger.WithError(err).Warnf("Import %s was done but couldn't be marked as such", i.ID)
+				s.logger.WithError(err).Warnf("Import %s was complete but couldn't be marked as such", i.ID)
 			}
+
+			key := fmt.Sprintf("%s.zip", i.TranslationID)
+			cfg, err := config.LoadDefaultConfig(context.TODO())
```

Comment:
I took a second look at this and decided that this implementation is superior than initializing the client when the ImportSupervisor is initialized, because creating the AWS client can return error, so `NewImportSupervisor` would also need to return an error, and then that would need to be handled by the caller.

Since the ImportSupervisor is created only once, this would mean ending program execution upon failure to contact AWS at startup time, when the program isn't ready to do anything with AWS

With the current implementation, since the config is created when the Supervisor is ready to actually use it, the program won't produce an error unless it can't communicate with AWS at the time it actually needs to, and the existing mechanisms for retrial will work when AWS stops producing errors from `LoadDefaultConfig`

So actually, while I tend to pretty readily accept feedback, I'd like to leave this how it is unless you feel strongly that I am wrong to do so.

Reply:
Additionally, I've introduced v2 of the AWS SDK here and I don't want to:

1) get bogged down or sidetracked with replacing v1 of the AWS SDK elsewhere in the project so that this config can be reused, as I don't see this as a high-value or priority task

2) rewrite this stanza with v1 so that it uses a v1 AWS Client that can be used elsewhere in the application, because it seems silly to add new code w/ an SDK that is being deprecated